runtime/lua/vim/_editor.lua

```diff
@@ -56,7 +56,6 @@ vim._extra = {
   inspect_pos = true,
 }
 
---- @private
 vim.log = {
```

Comment:
we don't have a logging module yet. that's why these aren't exposed.

if we are going to expose them they should be lowercased. the UPPERCASE names are pedantic and unusual for the nvim lua stdlib.

Reply:
These are already used all over the place in the plugin ecosystem for `vim.notify`

Reply:
I just updated the PR to use replace `@private` with `@nodoc`, per @lewis6991 's comment.